tools/plotting.py

```diff
@@ -178,6 +178,8 @@ def make_data_mc_comparison_plot( histograms = [],
         ax1 = plt.subplot( gs[1] )
         ax1.minorticks_on()
         ax1.grid( True, 'major', linewidth = 1 )
+        # Add horizontal line at y=1 on ratio plot
+        ax1.axhline(y=1, linewidth = 1)
```

Comment:
That will need some fixing later on.
Either as a function parameter or part of the HistogramProperties